Authentication/LogIn/View/LoginControllerDelegateType.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  LoginControllerDelegateType.swift
+//  Authentication
+//
+//  Created by Daniela Riesgo on 3/15/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import Foundation
+
+
+/**
+    Protocol gor login controller delegates.
+    Create your own delegate and override any of the defaut methods to
+    add behaviour to your login process.
+*/
+public protocol LoginControllerDelegateType {
```

Comment:
Delegates generally don't use the `Type` prefix. Rename it to `LoginControllerDelegate`.